virt/virt-4-8-release-notes.adoc

```diff
@@ -145,6 +145,47 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-8-known-issues"]
 == Known issues
 
+* If you initially deployed {VirtProductName} at version 2.4.z or earlier, upgrading to version 4.8 fails with the following message:
++
+[source,terminal]
+----
+risk of data loss updating hyperconvergeds.hco.kubevirt.io: new CRD removes version v1alpha1 that is listed as a stored version on the existing CRD
+----
++
+This bug does not affect clusters where {VirtProductName} was initially deployed at version 2.5.0 or later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986989[*BZ#1986989*])
++
+** As a workaround, remove the `v1alpha1` version and resume the upgrade process.
++
+. Run a proxy to the Kubernetes API server by running the following command:
++
+[source,terminal]
+----
+$ oc proxy &
+----
++
+. Remove the `v1alpha1` version from `.status.storedVersions` on the `HyperConverged` custom resource definition (CRD) by running the following command:
```

Comment:
You're replacing the API versions, so you might want to phrase this as "Replace the v1alpha1 API version with v1beta1 ..." 

"Remove" implies that something is being deleted.

Reply:
it's replacing `["v1alpha1", "v1beta1"]` with `["v1beta1"]` so it's implicitly removing `"v1alpha1"`

Reply:
While it is true that v1alpha1 is removed implicitly, we document what is happening explicitly. Unless "replace v1alpha1 with v1beta1" is incorrect, the explicit wording is normally the preferred option.

Reply:
@apinnick Simone's interpretation makes more sense to me due to the bug context:

> The issue is that:
> - CRD's status.storedVersions lists every version that has been a stored version (so in our case also v1alpha1 if CNV got initially installed at 2.4 time)
> - once that migration is complete, it's a manual process to remove unused storage versions from a CRD's .status.storedVersion.
> 
> HCO is currently correctly upgrading its CR to v1beta1 but, after that, it's not removing v1alpha1 from .status.storedVersion on the CRD.
> This prevents CNV to be upgraded to 4.8.0 that drops v1alpha1.